packages/frontend/src/components/AboutUsModal.tsx

```diff
@@ -0,0 +1,111 @@
+import {
+    Button,
+    Modal,
+    ModalBody,
+    ModalFooter,
+    ModalHeader,
+    ModalTitle,
+} from "@alphaday/ui-kit";
+
+interface IProps {
+    showModal: boolean;
+    onClose?: () => void;
+}
+export const AboutUsModal: React.FC<IProps> = ({ showModal, onClose }) => {
+    return (
+        <Modal showModal={showModal} onClose={onClose}>
```

Comment:
Please set modal size to "md" so that it doesn't look stretched.
<img width="743" alt="Screenshot 2024-08-08 at 21 29 53" src="https://github.com/user-attachments/assets/76e20e9d-6f3f-4329-84f7-308694b2fdb6">